### ISSUE TITLE
Necessary to allow to override the values of n_train_samples and n_valid_samples

### ISSUE THREAD

**Issue by AlessandroStaffolani** [ale8193/traffic-sign-recognition#8]
https://github.com/ale8193/traffic-sign-recognition/blob/27d25814161b5687d9901f46a70d622e2e0b9fd7/src/controllers/MenuController.py#L213-L215

It is necessary to allow in script mode to override the values of n_train_samples and n_valid_samples and also the split factor of training data, to allow to change the amount of data to use as train set and validation set
